src/test/fuzz/policy_estimator.cpp

```diff
@@ -41,7 +42,20 @@ FUZZ_TARGET(policy_estimator, .init = initialize_policy_estimator)
                     return;
                 }
                 const CTransaction tx{*mtx};
-                block_policy_estimator.processTransaction(ConsumeTxMemPoolEntry(fuzzed_data_provider, tx), fuzzed_data_provider.ConsumeBool());
+                const CTxMemPoolEntry entry = ConsumeTxMemPoolEntry(fuzzed_data_provider, tx);
+                NewMempoolTransactionInfo tx_info = {
```

Comment:
Nit: There is trailing whitespace here in commit 5ab08d23b2e88c1a8fda1535114e04b695c3483b. Also, why remove the const qualifier? Can't all the `tx_info` be made `const`?

Reply:
Thank you, Will update if there is need to re touch.